Frontend/src/index.css

```diff
@@ -12,18 +12,18 @@
   --font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
   --font-normal: 400;
   --font-bold: 700;
-  --base-font-size: 16px;
+  --base-font-size: 20px;
```

Comment:
pls avoid changes to existing props, pls create one with component prefix if required eg innerbox-base-font-size